tests/components/emulated_hue/test_hue_api.py

```diff
@@ -245,7 +246,7 @@ async def test_discover_lights(hue_client):
     result = await hue_client.get("/api/username/lights")
 
     assert result.status == HTTP_OK
-    assert "application/json" in result.headers["content-type"]
+    assert CONTENT_TYPE_JSON in result.headers["content-type"]
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6212145)